docs/docs/.vuepress/configs/sidebarConfig.js

```diff
@@ -17,6 +17,7 @@ const docsNav = [
       "/workflows/concurrency-and-throttling/",
       "/workflows/settings/",
       "/workflows/networking/",
+      "/workflows/workspaces/",
```

Comment:
I'd recommend keeping this at just `/workspaces` and merging with the content previously at `/organizations` (see comments below).

Reply:
It was originally structured this way, I defer to @psavkar